attesters/hydra-s1/index.ts

```diff
@@ -32,28 +39,17 @@ export const hydraS1SimpleAttester = async () =>
           name: "ZK Badge: Sismo Guest",
           description:
             "ZK Badge received by community members of frens of Sismo",
-          image: "./badges/sismo_guest.svg",
+          image: "./badges/badge_guest.svg",
           requirements: [],
         }),
       }),
       // Masquerade
       new AttestationsCollection({
         groups: [await Group.store.latest("sismo-masquerade-lens-followers")],
         badge: new Badge({
-          name: "Sismo Masquerade Bloomer ZK Badge",
-          description:
-            "ZK Badge owned by @masquerade.lens and @sismo.lens Lens followers",
-          image: "./badges/sismo_masquerade_bloomers.svg",
-          requirements: [],
-        }),
-      }),
-      // Ethereum-power-users
-      new AttestationsCollection({
```

Comment:
I think we are missing the Ethereum-power-users here?

Reply:
It's the code I wrote before the change to come into the main branch,I changed it